src/main/java/com/epam/page/object/generator/adapter/javaclass/IJavaClass.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.page.object.generator.adapter.javaclass;
+
+import com.epam.page.object.generator.adapter.annotation.IJavaAnnotation;
+import com.epam.page.object.generator.adapter.filed.IJavaField;
+import java.util.List;
+import javax.lang.model.element.Modifier;
+
+public interface IJavaClass {
```

Comment:
Please do not use "I" in a prefix in an interface name

Reply:
Done